language/diem-framework/modules/DiemAccount.move

```diff
@@ -1,4 +1,4 @@
-/// The `DiemAccount` module manages accounts. It defines the `DiemAccount` resource and
```

Comment:
I think it might be good to keep three backslashes here as the `docgen` might need it?

Reply:
I think the fix to this is the reason why unit test failed in CI, a `cargo run` in `diem-framework/` should update the related `.md` files and solve the problem.